dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNLccConverterInterfaceIIDM.cpp

```diff
@@ -60,7 +60,8 @@ LccConverterInterfaceIIDM::importStaticParameters() {
   staticParameters_.insert(std::make_pair("q", StaticParameter("q", StaticParameter::DOUBLE).setValue(-1 * getQ())));
   if (getBusInterface()) {
     double U0 = getBusInterface()->getV0();
-    double vNom = lccConverterIIDM_.getHvdcLine().get().getNominalV();
+    stdcxx::Reference<powsybl::iidm::HvdcLine> hvdcRef = lccConverterIIDM_.getHvdcLine();
+    double vNom = hvdcRef ? hvdcRef.get().getNominalV() : 0.0;
```

Comment:
This modifies the behavior of the test as before we assumed the hvdc existed

Reply:
Artefact of previous factorisation, required or DFL unit tests to work. Will be removed